src/modules/land_detector/land_detector_params_mc.c

```diff
@@ -84,3 +84,17 @@ PARAM_DEFINE_FLOAT(LNDMC_ROT_MAX, 20.0f);
  *
  */
 PARAM_DEFINE_FLOAT(LNDMC_ALT_MAX, -1.0f);
+
+/**
+ * Multicopter max distance to ground
+ *
+ * Maximum distance to ground allowed in the landed state (m/s up and down). Disabled if 0.
```

Comment:
> (m/s up and down)

?

Maybe also add that a range finder is required for this to work.

Reply:
Thanks, you caught my mindless param copypasta.